static/js/click_handlers.js

```diff
@@ -268,6 +268,18 @@ $(function () {
             subs.close();
         }
     });
+
+    $("#drafts_table").on("click", ".exit, #draft_overlay", function (e) {
+        if (meta.focusing) {
+            meta.focusing = false;
+            return;
+        }
+
+        if ($(e.target).is(".exit, .exit-sign, #draft_overlay, #draft_overlay > .flex")) {
+            drafts.close();
+        }
```

Comment:
Yeah, I think it works fine; in general we try to use preventDefault and friends in these handlers always, because like 50% of the time not having them is a bug, and having them unnecessarily doesn't really cause problems :).